src/Language/JavaScript/Parser/Grammar7.y

```diff
@@ -887,6 +897,7 @@ StatementNoEmpty : StatementBlock      { $1 {- 'StatementNoEmpty1' -} }
                  | ThrowStatement      { $1 {- 'StatementNoEmpty13' -} }
                  | TryStatement        { $1 {- 'StatementNoEmpty14' -} }
                  | DebuggerStatement   { $1 {- 'StatementNoEmpty15' -} }
+                 | ExportDeclaration   { $1 {- 'StatementNoEmpty16' -} }
```

Comment:
This doesn’t align with the ECMAScript 2015 spec but it seems the approach that #64 has taken:
https://github.com/erikd/language-javascript/pull/64/files#diff-843804451967a2b03f95fbef01a330d1R905

My first attempt created a separate `JSDeclaration` data type https://github.com/erikd/language-javascript/pull/79/commits/93308637a348958a32ff07e471b99d7552e03031 but then I reverted to align with the existing code.

Reply:
My maintenance of this has focused more on parsing real world JS rather than following the spec. This is fine.

Reply:
Thanks for clarifying. If not following the spec, how do you decide whether a program should pass or fail parsing? I thought most modern JS engines follow the spec but as I said, I don’t have a lot of background in this space.